src/current/_includes/releases/v23.2/v23.2.0-alpha.2.md

```diff
@@ -34,28 +58,49 @@ Release Date: October 2, 2023
 - A synthetic `dropped` column have been added to the `pg_attribute` table. This column tracks the attribution numbers for dropped attributions, to work around issues with ORMs that are not designed to handle gaps in attribution numbering in the `pg_attribute` table. [#111019][#111019]
 - Fixed a rare internal error in the `unnest` and `information_schema._pg_expandarray` [built-in functions](https://www.cockroachlabs.com/docs/v23.2/functions-and-operators) where passed string arguments could be cast to an array. [#110956][#110956]
 - External connection URLs now accept the scheme `azure-blob` for connections to Azure Blob Storage and the scheme `azure-kms` for connections to Azure KMS. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage. [#111217][#111217]
+- Fixed a bug where vectorized `COPY FROM` could produce a plan with more than one RenderNodes, when only zero or one should be allowed. This could result in multiple render nodes in a table with a hash shareded primary key. [#111284][#111284]
+- Fixed a bug in DB Console's Statement Diagnostic page that could cause the page to crash if the response was larger than 50 KB. The page now keeps pulling results until no maximum size errors are encountered. [#111128][#111128]
+- Fixed a bug where DB Console instances proxied at different subpaths that use OIDC pointed to an incorrect relative OIDC login path. [#111240][#111240]
+- Fixed a bug where changing the setting `server.telemetry.hot_ranges_stats.interval` had no effect. [#111305][#111305]
+
+<h3 id="v23-2-0-alpha-2-performance-improvements">Performance improvements</h3>
+
 - Fixed a performance bug that could result in rewriting a 128-MB file each time a store file is created, renamed, or removed when [Encryption At Rest](https://www.cockroachlabs.com/docs/v23.2/security-reference/encryption#encryption-at-rest) is enabled on a large store with many small files. [#111069][#111069]
+- Improved compaction heuristics to mitigate read amplification growth and admission control throttling when processing large deletes, such as during node decommissioning, replica rebalancing, or hen dropping tables. [#111277][#111277]
```

Comment:
@jbowens PTAL and make sure I've got this right.

Reply:
nit: "or when dropping"

otherwise LGTM